cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,59 @@
 /// <reference types='cypress' />
 
 describe('Book Store app', () => {
-  before(() => {
-    
+  beforeEach(() => {
```

Comment:
Consider creating a `cy.login()` command to login with API for better reusability and modularity.

Reply:
Done